app/views/patients/_fulfillment.html.erb

```diff
@@ -16,7 +16,7 @@
                           remote: true do |f| %>
     <div class="row">
       <div class="col">
-        <%= f.fields_for patient.fulfillment do |pt| %>
```

Comment:
do we expect these `patient.fulfillment` to be replaced everywhere after this PR? If so there are a few hanger-ons in the code.

Reply:
nah - it stops working in the form helpers, but actual data is still accessed via `patient.fulfillment.whatev`.